README.md

```diff
@@ -197,6 +204,43 @@ VCRCleaner::enable(array(
         body: '...response body...'
 ```
 
+### Post Field Content
+
+When making POST requests, you VCR will sometimes record the data inside of a `post_fields` parameter rather than the `body`. In those cases, this option can be used to sanitize sensitive content. Note that unlike the `body` field, `post_fields` is an array:
```

Comment:
```suggestion
When making POST requests, your VCR will sometimes record the data inside of a `post_fields` parameter rather than the `body`; e.g. when `CURLOPT_POSTFIELDS` is used in cURL. In those cases, this option can be used to sanitize sensitive content. Note that unlike the `body` field, `post_fields` is an array:
```

Since we know when `post_fields` can be used, we should give it as an example.

Reply:
Good call. I also did a little bit more research and it looks like it's mostly used when we don't set `CURLOPT_POST` to true. Here is the [documentation](forms.https://www.php.net/manual/en/function.curl-setopt.php):